### ISSUE TITLE
0.9.0 release

### ISSUE THREAD

**Issue by gaul** [kahing/catfs#47]
master has many fixes for modern Rust and improves compatibility with 32-bit and macOS.  It would be nice to pull in the fix for splice as well.

**Comment by jonringer** [comment]
seconded as well

**Comment by R0dri** [comment]
Any plans? Is this project maintained?

**Comment by shane-axiom** [comment]
I'd also like to get a sense of the status of this project, particularly:

| Catfs is ALPHA software. Don't use this if you value your data.

Is this statement still accurate?

**Comment by gaul** [comment]
I am helping @kahing for a few months and plan to release 0.9.0 later this month.  I have a few more issues to investigate but you can help out by testing master and filing issues.  As for the scary statement, I would apply that to all software; catfs is just more honest.

**Comment by shane-axiom** [comment]
Hah, yes, good point and understood, there are no guarantees. But there's a difference between "use at your own risk" and "we don't think this is at all ready." Sounds like it's approaching the former?

**Comment by gaul** [comment]
It is not my call to remove the warning but I think evaluating the open issues is a better way to evaluate quality.  If you find a reproducible bug that is something concrete we can work on.

**Comment by gaul** [comment]
@kahing Do I need to do anything other than tag a release or does this need to be uploaded to crates.io as well?

**Comment by ILOVEPIE** [comment]
Any updates?

**Comment by HenrikBengtsson** [comment]
I just found catfs the other day, and I must admit it took me quite a while to understand that the 0.8.0 release is dated, and that lots of things have been fixed in the master branch.  I'd like to pitch in with a few comments:

* Please consider bumping the version on master, so it can be distinguished from the 0.8.0 release, e.g. 0.8.0-9000 to indicate it's a development version based of 0.8.0.  That would help in bug reports when people output `catfs --version`.

* @gaul, if you think the master branch is ready for a release, then, yes, please consider releasing here on GitHub by add new tag 0.9.0. That way, folks have a new stable release to work with, and we can forget about 0.8.0. Although it would be nice to have it on crates.io, I think it would be unfortunate if lack of push privileges to crates.io would hold back a new release.

Thanks

**Comment by gaul** [comment]
Released 0.9.0: https://github.com/kahing/catfs/releases/tag/v0.9.0

I lack the karma to upgrade the crates.io binary.

**Comment by HenrikBengtsson** [comment]
Much appreciated.  Thank you.

For those how wants to build 0.9.0 from source, you can do:

```sh
git clone https://github.com/kahing/catfs.git
cd catfs
git checkout v0.9.0
mkdir /path/to/catfs-0.9.0
cargo install --root=/path/to/catfs-0.9.0 --path=.
```
so that

```sh
$ /path/to/catfs-0.9.0/bin/catfs --version
catfs 0.9.0
```
